### ISSUE TITLE
[Feature Request] Support for dark theme

### ISSUE THREAD

**Issue by nandgator** [vkohaupt/vokoscreenNG#211]
| [Cutefish DE](https://archlinux.org/packages/community/x86_64/fishui/) | [Xfce DE](https://archlinux.org/packages/extra/x86_64/xfdesktop/) |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/38415384/161033569-80b138a9-5631-4fca-90f9-aaac4f43be8f.png) | ![image](https://user-images.githubusercontent.com/38415384/161033593-0da3977a-03ab-4927-bb3b-5da09a3b79f6.png) |

Requesting support for dark theme.

**Comment by vkohaupt** [comment]
Last night I was working on a dark theme, I can't understand how anyone can do this. And when I went to bed I had chills, anxiety and nightmares.
What's worse, how are you supposed to work meaningfully with a dark theme? On my working environment, the Qt Creator, I see absolutely nothing. I see black writing or also blue or red writing on a black background. A compensation for pain and suffering is required for a dark theme.

Anyone who finds black humor may keep it. And if you still need a dark theme, you are welcome to make a big donation and I'll build a dark theme.

**Comment by nandgator** [comment]
Apparently vokoscreenNG picks up a dark theme neatly! And no, it's no [black magic](https://en.wikipedia.org/wiki/Black_magic).

![image](https://user-images.githubusercontent.com/38415384/185017608-e1d8a0d0-b911-4d27-b156-8600d78f5660.png)

This is what I did:

**`Method 1`** (Update: doesn't seem to work, see: https://github.com/vkohaupt/vokoscreenNG/issues/211?notification_referrer_id=NT_kwDOAkosGLMzMzY1NzExNjM1OjM4NDE1Mzg0#issuecomment-1218214779 - try `Method 2`)

1. Install [kvantum](https://archlinux.org/packages/community/x86_64/kvantum/)
2. [optional] Install any theme for kvantum <sup>[[1]](https://archlinux.org/packages/community/any/kvantum-theme-materia/) [[2]](https://aur.archlinux.org/packages?O=0&SeB=nd&K=kvantum-theme&outdated=&SB=p&SO=d&PP=50&submit=Go)</sup>.
3. Open kvantum theme manager and navigate to "Change/Delete Theme". Here you can choose any theme.

> OR

**`Method 2`**

1. Edit (or create) `~/.config/gtk-2.0/settings.ini` file. Paste the following lines:

    ```ini
    [Settings]
    gtk-application-prefer-dark-theme=true
    gtk-enable-animations=true # (optional)
    gtk-font-name="Inter 10" # (optional) your font name
    gtk-icon-theme-name="Tela-circle-manjaro-dark" # your icon theme name
    gtk-theme-name="Orchis-Green-Dark-Compact" # your gtk theme name
    ```

2. Create or a file called `~/.config/Trolltech.conf` and add the following content:

    ```conf
    [Qt]
    style=GTK+
    ```
3. Add these lines to `/etc/environment`

   ```env
   QT_QPA_PLATFORMTHEME=gtk2
   QT_STYLE_OVERRIDE=gtk2
   ```

4. Re-login / Restart

> Keeping this open for anyone who wants in-app theme switch.

**Comment by vkohaupt** [comment]
You're right, it's not black magic. Like every Qt program, vokoscreenNG also supports dark themes.
I understood Joe733 that he wants a built-in dark theme for desktops that don't support it.

Under KDE, or xfce with adwaita, a dark theme is set with two mouse clicks.

What desktop are you using?

**Comment by nandgator** [comment]
I'm using the XFCE DE. But I'm planning to switch to one of the WM.

**Update:** 
I've to mention this. When experimenting, I did `Method 2` first, then `Method 1`. Later I reverted `Method 2` and surprise, the dark theme was gone. So it looks like `Method 1` has no effect or I'm doing it the wrong way.

> I understood Joe733 that he wants a built-in dark theme for desktops that don't support it.

In my personal opinion, there should be a shared understanding between OS and user apps (here) on theming. The part - "should each OS/App developer enforce it" - is very subjective.

Closing this issue for now.

> _Unable to gaze the screen,_
> _due to the beloved vokoscreen,_
> _art thou much keen,_
> _for an in-app dark theme?_ 

Feel free to bump the thread, suggesting the issue to be re-opened
